lib/browser/rpc-server.js

```diff
@@ -465,6 +465,10 @@ ipcMain.on('ELECTRON_BROWSER_WINDOW_CONFIRM', function (event, message, title) {
 
 // Implements window.close()
 ipcMain.on('ELECTRON_BROWSER_WINDOW_CLOSE', function (event) {
-  event.sender.getOwnerBrowserWindow().close()
```

Comment:
I feel as though this should be a null check, looking through the rest of the file the other `try/catch` blocks are for potentially user influenced code.  Like `require` or calling functions or `global`.

This is just an edge case we need to handle I think with a null check (bug in code vs error that should be propagated).  Thoughts?